mucgpt-frontend/src/components/GlobalToastHandler/GlobalToastHandler.module.css

```
@@ -1,25 +1,66 @@
+    border-radius: 8px 0 0 8px;
+    z-index: 1;
+    pointer-events: none;
 }
 
-/* Icons */
-.successIcon,
-.errorIcon,
-.warningIcon,
-.infoIcon {
-    font-size: 16px;
+.toastWrapper :global(.fui-Toast) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)